src/target/drivers/display/8080/480x320x16_st7796.c

```diff
@@ -0,0 +1,149 @@
+/*
```

Comment:
this probably wants to be a .h file imported into 320x240x16.c (or we should adjust how it works/is named).  The DISPLAY_DRIVER code works best with a single file and HAS_LCD_TYPES, rather than duplicating the info twice...Maybe there is a better way